dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/dispatcher/internal/channel/HttpDispatcherLink.java

```diff
@@ -576,6 +576,7 @@ private void displayPage(InputStream inputStream, StatusCodes status) throws IOE
         try {
             if (exists(inputStream)) {
                 String theURI = this.isc.getRequest().getRequestURI();
+                this.response.setHeader(HttpHeaderKeys.HDR_CONTENT_SECURITY_POLICY.getName(), "default-src 'self'"); //OLGH31942 Adding CSP headers for the default welcome page
```

Comment:
This method is called from more than one location. Is this only adding the header for the default welcome page?